src/tools/rust-analyzer/crates/hir-expand/src/inert_attr_macro.rs

```diff
@@ -23,7 +23,7 @@ pub struct BuiltinAttribute {
 #[derive(Clone, Copy)]
 pub struct AttributeTemplate {
     pub word: bool,
-    pub list: Option<&'static str>,
+    pub list: Option<&'static [&'static str]>,
```

Comment:
This breaks code that rust-lang/rust CI doesn't check (src/tools/rust-analyzer/crates/ide-db/src/defs.rs)